### DISCUSSION TITLE
Invisible Notification on Github

### DISCUSSION THREAD

**Discussion by BlackAsLight** [community/community#49477] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So I seem to have somehow made one of my notification invisible on this page, and I don't know how to make it reappear or just that number next to Deno go away.
![image](https://user-images.githubusercontent.com/44320105/223640365-35756c58-2f89-44bd-8246-3460b4f26bfb.png)

**Comment by radar-blip** [comment]
Its a ghost notification.
See answer here: #48566 for fix.

**Comment by BlackAsLight** [comment]
Thanks

**Reply by radar-blip** [reply]
Anytime. Hope it helps!
